File: game-server/src/main/java/org/forfun/mmorpg/game/player/message/ResPlayerLogin.java
package org.forfun.mmorpg.game.player.message;

import jforgame.socket.share.annotation.MessageMeta;
import jforgame.socket.share.message.Message;
import org.forfun.mmorpg.game.player.service.PlayerService;

@MessageMeta(cmd = PlayerService.CMD_RES_LOGIN)
public class ResPlayerLogin implements Message {

	private byte status;

	public byte getStatus() {
		return status;
	}

	public void setStatus(byte status) {
		this.status = status;
	}

}

File: game-server/src/main/java/org/forfun/mmorpg/game/player/message/ResPlayerLogin.kt
package org.forfun.mmorpg.game.player.message

import jforgame.socket.share.annotation.MessageMeta
import jforgame.socket.share.message.Message
import org.forfun.mmorpg.game.player.service.PlayerService

@MessageMeta(cmd = PlayerService.CMD_RES_LOGIN)
class ResPlayerLogin : Message {

    var status: Byte = 0
}
